src/pages/Home.test.js

```diff
@@ -0,0 +1,13 @@
+// import {screen, render} from '@testing-library/react';
+// import Home from './Home';
+
+// describe ("<Home/>", () => {
+//     test("render the home page Jason", () => {
+//         render(<Home/>)
+        
+//         const element = screen.getByText(/Welcome to Cat Tinder/i)
+        
+
+//        expect(element).toBeInTheDocument()
+//     })
+// })
```

Comment:
Remove this file and bring it back for the next branch. best to not have it on the main branch even as a comment. Feel free to store it somewhere else.